packages/block-editor/src/hooks/background.js

```diff
@@ -134,10 +138,25 @@ export function BackgroundImagePanel( {
 	setAttributes,
 	settings,
 } ) {
-	const style = useSelect(
-		( select ) =>
-			select( blockEditorStore ).getBlockAttributes( clientId )?.style,
-		[ clientId ]
+	const { style, inheritedValue, _links } = useSelect(
+		( select ) => {
+			const { getBlockAttributes, getSettings } =
+				select( blockEditorStore );
+			const _settings = getSettings();
+			return {
+				style: getBlockAttributes( clientId )?.style,
+				_links: _settings[ globalStylesLinksDataKey ],
+				/*
+				 * @TODO 1. Pass inherited value down to all block style controls,
+				 *   See: packages/block-editor/src/hooks/style.js
+				 * @TODO 2. Add support for block style variations,
+				 *   See implementation: packages/block-editor/src/hooks/block-style-variation.js
```

Comment:
So if a variation has been applied to a block, we need to parse it here. 

This needs to happen in a follow up. I've added a note to https://github.com/WordPress/gutenberg/issues/54336

cc @aaronrobertshaw just in case there are gotchas (TL;DR - I'm trying to pass down the theme styles applied to a block to the controls, so that the controls are aware of any defaults)

Reply:
Yeah, at a glance, we'd need to detect which, if any, variation has been applied to the block, then merge the variation's data over the block type's data. The resulting set would tell us which value the control should get.